### PR TITLE
feat(widget-message-meet): add ability to display login form

### PR DESCRIPTION
**Pull request by adamweeks**
This will be used in demos and will hopefully have the OAuth functionality added to it as well.

### CODE REVIEW

**Comment by adamweeks** [comment]
Login screen:
screenshot 2016-12-14 14 44 02

**Comment by adamweeks** [comment]
Maybe we need to change the button text to "Go" instead of "Chat"?

File: packages/widget-message-meet/src/selectors/activities.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export const getMostRecentReadReceipts = createSelector(
+  [getActivities, getParticipants],
+  (activities, participants) => {
+    const activity = _.last(activities);
+    return participants.filter((participant) =>
+      participant.roomProperties.lastSeenActivityUUID === activity.id
+    );
+  }
+);
```
Comment (bzang) [inline packages/widget-message-meet/src/selectors/activities.js]: Were did this file come from? Doesn't seem like it's supposed to be a part of this PR.

File: packages/widget-message-meet/src/components/widget-data-input/index.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    const {accessToken, toPersonEmail} = props;
+    const error = false;
+    this.state = {
+      accessToken,
+      error,
+      toPersonEmail
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.handleAccessTokenChange = (e) => this.setState({accessToken: e.target.value});
+    this.handleEmailChange = (e) => this.setState({toPersonEmail: e.target.value});
```
Comment (bzang) [inline packages/widget-message-meet/src/components/widget-data-input/index.js]: Not the optimal way to do this, but I'm guessing this is just a stop gap until we get OAuth working.

File: packages/widget-message-meet/src/selectors/activities.js (hunk 1 above)
Comment (adamweeks) [inline packages/widget-message-meet/src/selectors/activities.js]: Yep, it was from my 15 mins of actual coding yesterday, lol.

File: packages/widget-message-meet/src/components/widget-data-input/index.js (hunk 2 above)
Comment (bzang) [inline packages/widget-message-meet/src/components/widget-data-input/index.js]: Just setting it into the redux store instead of just using set state.

Comment (adamweeks) [inline packages/widget-message-meet/src/components/widget-data-input/index.js]: Oh yea, I didn't want to pollute the store with something that will probably only be used with demos.

**Comment by adamweeks** [comment]
Will reopen later.

**Pull request closed without merging**